internal/action/handle.go

```diff
@@ -81,15 +77,15 @@ func DelRegistry(sources *registry.RegistryDataSource, args []string) int {
 	_, ok := sources.Registry[name]
 
 	if !ok {
-		logger.PrintError("[Grm]: can't found alias" + " " + name + " " + "in your .nrmrc file. Please check it exist.")
+		logger.PrintError("[Grm]: can't found alias" + " " + name + " " + "in your .nrmrc file. Please check it exist." + registry.Eol())
 		return 1
 	}
 	flag, err := registry.DelNrm(name)
 	if flag {
-		logger.PrintSuccess("[Grm]: del registry" + " " + name + " " + "success!")
+		logger.PrintSuccess("[Grm]: del registry" + " " + name + " " + "success!" + registry.Eol())
 		return 0
 	}
-	logger.PrintError("[Grm]: del registry fail " + err.Error())
+	logger.PrintError("[Grm]: del registry fail " + err.Error() + registry.Eol())
```

Comment:
I noticed that almost all the `print` functions use the `Eol()` function, are we considering integrating `eol()` into the `print` function?

Reply:
@zaunist I think logger module shouldn't change. may we add new func to do this? A method  to combse messages

Reply:
> @zaunist I think logger module shouldn't change. may we add new func to do this? A method to combse messages

Agree with you.